app/views/users/edit.html.slim

```diff
@@ -50,28 +50,29 @@
   = f.input :company_name, label: 'Company name'
   = f.input :company_info, label: 'Company info', hint: 'Where is the office, how much space can you provide, etc.'
   = f.input :bio, label: 'About you', hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
+
+  h3.page-header Technical Skills:
+  = f.input :tags, wrapper_html: { class: 'col-md-6' }, placeholder: 'Languages, techniques, etc involved', label: 'Technical_Expertise', hint: 'Separate tags by comma.'
```

Comment:
`tags` and `tags1` are not the best choices for a name. Can you think of a more descriptive, intention-revealing one?